src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java

```diff
@@ -199,7 +199,7 @@ public String getDisplayName() {
     private static AssumeRoleRequest createAssumeRoleRequest(String iamRoleArn) {
         return new AssumeRoleRequest()
                 .withRoleArn(iamRoleArn)
-                .withRoleSessionName(Jenkins.getActiveInstance().getDisplayName());
+                .withRoleSessionName(Jenkins.getActiveInstance().getDisplayName().trim());
```

Comment:
There is no reason to localize this at all.

```suggestion
                .withRoleSessionName("Jenkins");
```

Reply:
Up to maintainers, see the PR description

Reply:
AFAIK we just need to pass _something_ as the role session name, it does not really matter what; and since we cannot guarantee that a localization will match `[\w+=,.@-]*` even after trimming, it is not safe to use `Jenkins.displayName` at all.

Reply:
I agree, there is no reason for using the localized name but something as Jesse says.